js/main.js

```diff
@@ -678,65 +678,44 @@ function DragAndDropTable(table,mode,queryKey) {
 	 */
 	function FullSizeImage(els) {
 		if (!els) return;
-		els = (typeof els == "object" || typeof els == "function") && typeof els.length == "number"?els:[els];
-		var hashTrigger = null;
-		var body = document.body;
-   		var imageCanvas = document.getElementById("image-canvas") || document.createElementWithAttributes("div", {"id": "image-canvas"}, body);
-		imageCanvas.setVisible = function(visible) {
-			if (visible)
-				this.classList.remove("js-display-none");
-			else
-				this.classList.add("js-display-none");
-		};
-		var stopTrigger = function() {
-			if (hashTrigger) {
-				window.clearInterval(hashTrigger);
-				var scrollPos = document.getScrollPosition();
-				window.history.back();
-				// Fuer den Fall, dass man bei eingeblendeten Bild gescrollt hat
-				window.scrollTo(scrollPos.left, scrollPos.top);
-			}
-		};
+		const isPopoverSupported = () => HTMLElement.prototype.hasOwnProperty("popover");
```

Comment:
I doubt that this line does what you had in mind. From my point of view, `isPopoverSupported` is a function and the conditions checks whether the function exists or not, i.e., 

```
 const isPopoverSupported = () => HTMLElement.prototype.hasOwnProperty("FooBar");
 if (isPopoverSupported) // always true because the function exists (not the property)
```

 `if (!HTMLElement.prototype.hasOwnProperty("popover"))` appears to be sufficient

Reply:
I rewrote it as you proposed in commit [120c8b6](https://github.com/My-Little-Forum/mylittleforum/pull/838/commits/120c8b67a4da7545f7b5adead7765c40ed60ee43). This solves this and the comment above.